scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/JobData.java

```diff
@@ -98,6 +98,54 @@
                 @NamedQuery(name = "getMeanJobPendingTime", query = "select avg(startTime - submittedTime) from JobData where startTime > 0 and submittedTime > 0"),
                 @NamedQuery(name = "getMeanJobSubmittingPeriod", query = "select count(*), min(submittedTime), max(submittedTime) from JobData"),
                 @NamedQuery(name = "getTotalJobsCount", query = "select count(*) from JobData"),
+                @NamedQuery(name = "getJobsNumberWithStatusAndNoTenant", query = "select count(*) from JobData where status in (:status) and tenant = null"),
```

Comment:
There are too many similar named queries. It is much more elegant to use the CriteriaApi to build queries dynamically